lynx/src/main/java/com/github/pedrovgs/lynx/LynxConfig.java

```diff
@@ -55,10 +58,21 @@ public LynxConfig setMaxNumberOfTracesToShow(int maxNumberOfTracesToShow) {
   }
 
   public LynxConfig setFilter(String filter) {
+    if (filter == null) {
+      throw new NullPointerException("filter can't be null");
+    }
     this.filter = filter;
     return this;
   }
 
+  public LynxConfig setFilterTraceLevel(TraceLevel filterTraceLevel) {
+    if (filterTraceLevel == null) {
+      throw new NullPointerException("filterTraceLevel can't be null");
+    }
```

Comment:
Same here:


public LynxConfig setFilterTraceLevel(TraceLevel filterTraceLevel) {
    if(filterTraceLevel == null){
        throw new IllegalArgumentException("You can't use a null instance as filterTraceLevel");
    }
}

```